shortcodes/ucf-degree-list-shortcode.php

```diff
@@ -28,7 +29,21 @@ public static function shortcode( $atts ) {
 				'order'          => 'ASC'
 			);
 
-			if ( $atts['filter_by_tax'] && $atts['terms'] ) {
+			if ( $atts['filter_by_taxonomies'] && $atts['terms'] ) {
+				$taxonomies = explode( ' ', $atts['filter_by_taxonomies'] );
```

Comment:
Look into possibly using dynamic attributes to make this a little more verifiable/explicit for users. I.e. if filtering by college and program_types on degrees, using `[ucf-degree-list ... colleges_terms="term1,term2" program_types_terms="term1, term2"]` would make it more obvious... at that point the `filter_by_tax` could be used and accept a comma delimited list of taxonomies, if we want to be even more explicit.